static/app/views/dashboardsV2/view.tsx

```diff
@@ -23,6 +24,13 @@ type Props = RouteComponentProps<{orgId: string; dashboardId: string}, {}> & {
 };
 
 function ViewEditDashboard(props: Props) {
+  useEffect(() => {
+    const {api, organization, params} = props;
+    if (params.dashboardId && params.dashboardId !== 'default-overview') {
+      updateDashboardVisit(api, organization.slug, params.dashboardId);
+    }
+  });
```

Comment:
You'll want to specify a dependency array here to avoid this hook being called on every render.

Reply:
This PR needs to be updated, it's stale 1 sec

Reply:
Updated @Zylphrex And the thing you mentioned was updated in the previous [PR](https://github.com/getsentry/sentry/blob/master/static/app/views/dashboardsV2/view.tsx#L35)